manifests/a/AVerMedia/RECentral/4.7.44.1/AVerMedia.RECentral.locale.en-US.yaml

```
@@ -0,0 +1,14 @@
+# Created using wingetcreate 1.1.2.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.2.0.schema.json
+
+PackageIdentifier: AVerMedia.RECentral

```

```suggestion
PackageIdentifier: AVerMedia.RECentral.4
```
The `4` Should be added as it is part of the Display Name, and to prevent conflicts with AVerMedia RECentral Express in the future